addons/website/static/src/components/views/theme_preview.xml

```diff
@@ -1,30 +1,45 @@
 <?xml version="1.0" encoding="utf-8"?>
 <templates id="template" xml:space="preserve">
     <t t-name="website.ThemePreviewForm.ControlPanel" t-inherit="web.ControlPanel" t-inherit-mode="primary">
+        <xpath expr="//*[hasclass('o_control_panel')]" position="attributes">
+            <attribute name="class" remove="pb-3" add="pb-2" separator=" "/>
+        </xpath>
+        <xpath expr="//*[hasclass('o_control_panel_breadcrumbs')]" position="attributes">
+            <attribute name="class" add="flex-grow-1" separator=" "/>
+        </xpath>
+        <xpath expr="//*[hasclass('o_control_panel_main_buttons')]" position="inside">
+            <button class="d-none d-md-inline-block btn btn-secondary o_switch_theme me-1" t-on-click.stop="this.back">
```

Comment:
Since you added some classes, I'd reorder the whole line, even if it was not coming from you.

```suggestion
            <button class="o_switch_theme btn btn-secondary d-none d-md-inline-block me-1" t-on-click.stop="this.back">
```